ansible/roles/openshift_setup_ssl/tasks/main.yml

```diff
@@ -0,0 +1,310 @@
+---
+  - set_fact:
+      openshift_build_type: "mac"
+      kube_build_type: "darwin"
+      openshift_release_ext: "zip"
+    when: ansible_os_family == "Darwin"
+
+  - set_fact:
+      openshift_build_type: "linux-64bit"
+      kube_build_type: "linux"
+      openshift_release_ext: "tar.gz"
+    when: ansible_os_family == "RedHat"
+
+  - set_fact:
+      oc_tools_dir: /usr/bin
```

Comment:
oc_tools_dir needs to be in the user's path for the oc commands to execute correctly.  The other option would be to execute oc commands with a modified environment.PATH variable.